.github/workflows/main.yml

```
@@ -25,7 +25,7 @@ jobs:
     env:
       CONDA_ENV: 'test_env'
       VERSION_PYTHON: ${{ matrix.version_python }}
-      BENCHOPT_BRANCH: benchopt:main
+      BENCHOPT_BRANCH: tommoral:ENH_parallelization
```

is this needed for this PR or committed by accident ?

---

This is a mechanism to run the test with a branch that is not `main`. As this PR depends on this change, this is intentionnal.

---

As benchopt/benchopt#255 has been merged, we can now revert this
```suggestion
      BENCHOPT_BRANCH: benchopt:main
```